tests/test_requirements/google_tests_requirements.txt

```diff
@@ -0,0 +1,4 @@
+google-api-python-client
+google-cloud-storage
+gcsfs
+fsspec<=2023.5.0
```

Comment:
fsspec is already in base reqs.txt, and I think we can remove this altogether since it's a requirement of runhouse already